ocs_samples/basic_samples/Dataviews/Python3/program.py

```diff
@@ -226,20 +223,22 @@ def main(test=False):
         print
         print("Getting dataview")
         dv = ocsClient.Dataviews.getDataview(namespaceId, sampleDataviewId)
-
+        print(dv.toJson())
         # assert is added to make sure we get back what we are expecting
-        expectedJSON = '{"Id": "Dataview_Sample", "Queries": [{"Id": "Dataview_Sample", "Query": {"Resource": "Streams", "Field": "Name", "Value": "SampleStream", "Function": "Contains"}}], "Name": "Dataview_Sample_Name", "Description": "A Sample Description that describes that this Dataview is just used for our sample.", "Mappings": {"IsDefault": true, "Columns": [{"Name": "time", "IsKey": true, "DataType": "DateTime", "MappingRule": {"PropertyPaths": ["time"]}}, {"Name": "DefaultGroupRule_Tags", "IsKey": false, "DataType": "string", "MappingRule": {"GroupRuleId": "DefaultGroupRule", "GroupRuleToken": "Tags"}}, {"Name": "pressure", "IsKey": false, "DataType": "Double", "MappingRule": {"PropertyPaths": ["pressure"]}}, {"Name": "temperature", "IsKey": false, "DataType": "Double", "MappingRule": {"PropertyPaths": ["temperature"]}}]}, "IndexDataType": "datetime", "GroupRules": [{"Id": "DefaultGroupRule", "Type": "StreamTag", "TokenRules": null}]}'
+        startIndex = indexConfigObj.StartIndex + ":00Z"
+        endIndex = indexConfigObj.EndIndex + ":00Z"
+        expectedJSON = '{"Id": "Dataview_Sample", "Queries": [{"Id": "Dataview_Sample", "Query": "name:*SampleStream*"}], "Name": "Dataview_Sample_Name", "Description": "A Sample Description that describes that this Dataview is just used for our sample.", "Mappings": {"IsDefault": true, "Columns": [{"Name": "time", "IsKey": true, "DataType": "DateTime", "MappingRule": {"PropertyPaths": ["time"]}}, {"Name": "pressure", "IsKey": false, "DataType": "Double", "MappingRule": {"PropertyPaths": ["pressure"]}}, {"Name": "temperature", "IsKey": false, "DataType": "Double", "MappingRule": {"PropertyPaths": ["temperature"]}}]}, "IndexConfig": ' + \
+            '{"StartIndex": ' + f'"{startIndex}"' + ', "EndIndex": ' + f'"{endIndex}"' + \
```

Comment:
We should do this better

Reply:
as in just one string here

Reply:
Yeah, I struggled a bit to simplify since because of IndexConfig it's not a static string anymore. Maybe a job for .toJson() on IndexConfig to return the right thing?

Reply:
seems good.  its a big pain, but I think its worthwhile (at least until after Dataviews are out of beta to see if the structure changed).